### PR TITLE
Profile Page Prototype

### PR DESCRIPTION
**Pull request by jayyduran0108**
I created the ProfileServlet which grabs the username the user clicked on. I create the profile.jsp which verifies if the user is looking at their own profile or not.

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<<%--
```
Comment (chilimalone) [inline src/main/webapp/WEB-INF/view/profile.jsp]: extra <

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      response.sendRedirect("/login");
+      return;
+    }
+
+    String requestUrl = request.getRequestURI();
+    String profile = requestUrl.substring("/users/".length());
+
+    request.setAttribute("profile",profile);
+    
+  /** Extract the URL
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: remove dead code before you submit.

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+public class ProfileServlet extends HttpServlet {
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: add new line before the class declaration

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.List;
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: add newline before this to separate different library imports

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 5]
```diff
@@ -0,0 +1,96 @@
+package codeu.controller;
+
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: nit: add newline before to separate library imports (for readability)

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+import java.util.UUID;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+public class ProfileServlet extends HttpServlet {
+
+  /** Store class that gives access to Users. */
```
Comment (chilimalone) [inline src/main/java/codeu/controller/ProfileServlet.java]: I think the existing code all uses 4 spaces for indent. Please use 4 spaces for consistency :)

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  Copyright 2017 Google Inc.
```
Comment (chilimalone) [inline src/main/webapp/WEB-INF/view/profile.jsp]: edit date?

**Review by chilimalone — CHANGES_REQUESTED** [review]
Good job! Please wait for reviews before merging your code in.  I've just a few nits

File: src/main/webapp/WEB-INF/view/profile.jsp (hunk 1 above)
Comment (jayyduran0108) [reply src/main/webapp/WEB-INF/view/profile.jsp]: done

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 2 above)
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 3 above)
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 4 above)
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 5 above)
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/java/codeu/controller/ProfileServlet.java (hunk 6 above)
Comment (jayyduran0108) [reply src/main/java/codeu/controller/ProfileServlet.java]: done

File: src/main/webapp/WEB-INF/view/profile.jsp (hunk 7 above)
Comment (jayyduran0108) [reply src/main/webapp/WEB-INF/view/profile.jsp]: done

**Comment by chilimalone** [comment]
Hey, thanks for responding to my comments! Please push your changes to the branch and let me know so I can confirm that the changes are present and approve the merge :)

**Review by annepham25 — APPROVED** [review]

**Pull request merged**